client/index.html

```diff
@@ -1,10 +1,16 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="fr">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>JS Monorepo</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
It's better to import font files, to serve them directly. In .woff2 extension, for example.

Reply:
correction done